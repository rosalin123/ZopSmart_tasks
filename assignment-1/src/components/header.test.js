import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import Header from './header';

const mockStore = configureMockStore();
const store = mockStore({});

let wrapped = shallow(
  <Provider store={store}>
    <Header />
  </Provider>
);

describe('Header Component', () => {
  it('Header should render without throwing an error', () => {
    expect(wrapped).toMatchSnapshot();
  });
});
