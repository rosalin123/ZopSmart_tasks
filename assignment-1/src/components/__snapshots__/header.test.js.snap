// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Header should render without throwing an error 1`] = `ShallowWrapper {}`;
